src/link/Coff.zig

```diff
@@ -1034,6 +1038,10 @@ fn linkWithLLD(self: *Coff, comp: *Compilation) !void {
             try argv.append(p);
         }
 
+        if (self.base.options.out_implib) |out_implib| {
```

Comment:
There's an `unused capture` error on this line; change to `|_|`.

Reply:
Fixed, thanks.